library/core/src/main/java/com/google/android/exoplayer2/Format.java

```diff
@@ -92,6 +92,10 @@
 
   // Video specific.
 
+  /**
+   * Codec specific profile value or {@link #NO_VALUE} if unknown or not applicable.
+   */
+  public final int profile;
```

Comment:
In other cases we have signaled the profile in `Format.codecs` as an RFC 6381 codecs string. [`MediaCodecUtil.getCodecProfileAndLevel`](https://github.com/google/ExoPlayer/blob/34d0e0c/library/core/src/main/java/com/google/android/exoplayer2/mediacodec/MediaCodecUtil.java#L205) parses this. Could you try switching to that way of signaling the profile, as it would avoid having two ways to do this and reduce the number of files that need to be modified?

Reply:
Ok. I will change the signaling